Cambio Cubierta Archicad

CristianAnchali opened the pull request:
He creado una viga longitudinal adicional, con la finalidad de lograr mayor rigidez en el elemento. He notado que la cubierta alcanza casi los 10m, por lo que me parece prudente agregar esta viga. Espero que este aporte sea de ayuda.

CristianAnchali committed:
Cambio Cubierta Archicad

Cambios Cubierta

d10genez committed:
Actualización errores

Se observo que el primer párrafo se repetía tres veces

d10genez commented:
En la construcción del Tena se aplicó esta solución, ya que cada viga de cubierta fue de hasta 9,5m. Varió ya que el cumbrero tuvo una forma curva.

d10genez committed:
Modelado 3D en Blender

d10genez committed:
Se agrega fuentes de blender y EnergyPlus

d10genez committed:
Error

d10genez committed:
Simulaciones EnergyPlus

d10genez committed:
Modelado 3D en FreeCAD

d10genez committed:
Plantillas para IDF (EnergyPlus)

Tutoriales y plantillas con parámetros base y diccionario de datos estándar de EnergyPlus

d10genez committed:
Archivos de verificación estructural en bambú

SAP, EXCEL 
Propiedades
Tablas NEC
Formulas 
Verificación

d10genez committed:
Verificación bioclimática y energética

EPW Ecuador
Simulación Lectura y comportamiento
Simulación de Cliclos de costo HVAC
tolita-bamboo.idf
Euclid

d10genez committed:
Modelo 3D 

Blender

d10genez committed:
Nueva versión en proceso.

Se entrega modelo Blender 3D con cumbreras y columnas principales reforzadas.

d10genez committed:
Se entrega modelo 3D Blender

Se entrega modelo Blender 3D con cumbreras y columnas principales reforzadas. Última corrida del modelo matemático.

d10genez committed:
Se entrega modelo 3D Blender

Se entrega modelo Blender 3D con cumbreras y columnas principales reforzadas. Última corrida del modelo matemático.

d10genez committed:
Se entrega modelo 3D en Blender

Se entrega modelo Blender 3D con cumbreras y columnas principales reforzadas. Última corrida del modelo matemático.

d10genez committed:
Se entrega modelo 3D blender

Blender 3D con cumbreras y columnas principales reforzadas. Última corrida del modelo matemático.

d10genez committed:
Se migra a herramientas abiertas

Se agrega modelo 3D en Blender

d10genez committed:
Se Agregan formatos de formato abierto

FreeCAD, Blender

d10genez committed:
Se entrega modelo Blender 3D

Nuevo modelo 3D Blender, con refuerzo en cumbreras y columnas principales.

d10genez committed:
Nueva versión 

Se asignan un solo material y agregar criterios estructurales

d10genez committed:
Merge pull request #9 from d10genez/master-final

Modelo luego del análisis estructura matemático.

d10genez committed:
Nueva imagen intro

Visualizar lo maleable de la estructura.

d10genez committed:
Descontinuado

Se ha consolidado todos los archivos EnergyPlus en la carpeta tolita-bamboo-bioclimamtica

d10genez committed:
Descontinuado

Revisar la carpeta tolita-bamboo-bioclimatica

d10genez committed:
Descontinuado

Se está centralizando todo en aplicaciones open sources: tolita-bamboo-mat.blend, TOLITA.FREECAD.FCStd

d10genez committed:
Descontinuado

Revisar la carpeta tolita-bamboo-bioclimatica

d10genez committed:
Update README.md

d10genez committed:
Merge pull request #10 from d10genez/master-final

Master 2.0

d10genez committed:
Nueva imagen intro

d10genez committed:
Nuevo Modelo 

Modelo incluye tres tipos de adaptación.

d10genez committed:
Merge pull request #11 from d10genez/master-final

Nuevo Modelo 3D Blender
